packages/m/mootable-hashmap.json

```
@@ -0,0 +1,34 @@
+{
+  "description": "Full featured, fast, HAMT based HashMaps",
+  "filename": "index.js",
+  "homepage": "https://mootable.github.io/hashmap/",
+  "keywords": [
+    "map",
+    "hashmap",
+    "linkedhashmap",
+    "hamt"
+  ],
+  "license": "MIT",
+  "name": "mootable-hashmap",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mootable/hashmap.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "@mootable/hashmap",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*.@(js|map)"
```

```suggestion
          "*.@(js|map)"
```
It looks like all the files are just in this directory, so let's avoid a globstar when we can.